cloudbridge/cloud/providers/azure/provider.py

```diff
@@ -54,8 +54,8 @@ def __init__(self, config):
                                str(self.resource_group)))[-6:]))
 
         self.vm_default_user_name = self._get_config_value(
-            'azure_vm_default_user_name', get_env(
-                'AZURE_VM_DEFAULT_USER_NAME', 'cbuser'))
+            'azure_vm_default_username', get_env(
```

Comment:
Didn't we agree that this should accept both versions for backward compatibility, with the `azure_vm_default_user_name` being deprecated?

Reply:
Added in https://github.com/CloudVE/cloudbridge/pull/152/commits/742ca14de557f6f34c449b7fa2b5374a78f2bf2d